src/css/fss-reset-global.css

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
@jobara - I had thought that the FSS was deprecated and removed from Infusion a while ago, in favour of industry-standard solutions such as Foundation as per https://issues.fluidproject.org/browse/FLUID-5469 - I assume that these should not be here?